### PR TITLE
Added 3.11 Curated Environment for 22.04

### PR DESCRIPTION
**Pull request by v-kaushal**

### CODE REVIEW

File: assets/inference/environments/minimal-ubuntu22.04-py311-cpu-inference/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+$schema: https://azuremlschemas.azureedge.net/latest/environment.schema.json
+
+description: >-
+  AzureML minimal/Ubuntu 20.04/Python 3.9 cpu environment.
```
Comment (vizhur) [inline assets/inference/environments/minimal-ubuntu22.04-py311-cpu-inference/spec.yaml:4]: ubuntu 22?

Comment (v-kaushal) [reply assets/inference/environments/minimal-ubuntu22.04-py311-cpu-inference/spec.yaml]: Hi vizhur, 
I made changes in the new commit for the above.

**Review by vizhur — APPROVED** [review]

File: assets/inference/environments/minimal-ubuntu22.04-py311-cpu-inference/context/conda_dependencies.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: minimal
+channels:
+- anaconda
+- conda-forge
+dependencies:
+- python=3.11
+- pip=22.1.2
+- pip:
+  - azureml-inference-server-http=={{latest-pypi-version}}
+  # vulnerabilities
```
[the remark this answers is not in the document]
Comment (v-kaushal) [reply assets/inference/environments/minimal-ubuntu22.04-py311-cpu-inference/context/conda_dependencies.yaml:10]: removed packages below # vulnerabilities and hence resolving this comment.

File: assets/inference/environments/minimal-ubuntu22.04-py311-cpu-inference/context/conda_dependencies.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+name: minimal
+channels:
+- anaconda
+- conda-forge
```
[the remark this answers is not in the document]
Comment (v-kaushal) [reply assets/inference/environments/minimal-ubuntu22.04-py311-cpu-inference/context/conda_dependencies.yaml:3]: Made `conda-forge` as the first channel for conda to look for

**Pull request merged**